src/app/pages/common/entity/entity-form/components/form-errors/form-errors.component.html

```diff
@@ -12,7 +12,7 @@
         {{"Value must be greater than " | translate}} {{control.errors.fields[0]}}.
     </mat-error>
     <mat-error *ngIf="control.hasError('forbidden')">
-        {{"This name has already been used." | translate}}
+        {{"This name " | translate}} <i>{{control.errors.value}}</i> {{"is alreay in use." | translate}}
```

Comment:
Lets word this like how groups has it "The name" "is already in use". (also you misspelled already)